bundle/manifests/search-v2-operator.clusterserviceversion.yaml

```diff
@@ -227,13 +227,13 @@ spec:
                     name: manager
                     env:
                       - name: POSTGRES_IMAGE
-                        value: registry.redhat.io/rhel8/postgresql-13:1-56
+                        value: registry.redhat.io/rhel8/postgresql-13@sha256:d8ec037e60575b86320fdc162117600240ce53038fb8b9b7296ce525d2b8e605
```

Comment:
@jlpadilla are we updating the Postgres version?

Reply:
Yes, this will update to `1-72`
It's recommended to use SHA vs. tag so I'm trying to switch to use the sha.